api/src/Controller/Announcement/ListMemberAnnouncements.php

```diff
@@ -66,66 +67,30 @@ public function buildResource(Request $request, Response $response, $args): arra
     {
         $user = $this->findMemberId($request, $response, $args, 'id');
         $user = $user['id'];
-        $sth = $this->container->db->prepare(<<<SQL
-            SELECT
-                *
-            FROM
-                `Announcements`
-            WHERE
-                `Scope` = 0 OR
-                `Scope` = 1 AND (
-                    SELECT
-                        COUNT(AccountID)
-                    FROM
-                        `ConComList`
-                    WHERE
-                        `AccountID`  = '$user'
-                ) > 0 OR
-                `Scope` = 2 AND (
-                `DepartmentID` IN(
-                SELECT
-                    `DepartmentID`
-                FROM
-                    `ConComList`
-                WHERE
-                    `AccountID` = '$user'
-            ) OR `DepartmentID` IN(
-                SELECT
-                    `DepartmentID`
-                FROM
-                    `Departments`
-                WHERE
-                    `ParentDepartmentID` IN(
-                    SELECT
-                        `DepartmentID`
-                    FROM
-                        `ConComList`
-                    WHERE
-                        `AccountID` = '$user'
-                )
-            ))
-            ORDER BY `PostedOn` ASC
-SQL
-        );
-        $sth->execute();
-        $todos = $sth->fetchAll();
+        $select = Select::new($this->container->db);
+
+        $sub1 = $select->subselect()->columns('COUNT(AccountID)')->from('ConComList')->whereEquals(['AccountID' => $user]);
```

Comment:
This is where I want us to fix our phpcs rules (once I figure out exactly how) to allow hanging arrows. There has to be a way. I'll find it eventually.